invenio_communities/communities/resources/ui_schema.py

```diff
@@ -36,6 +36,7 @@ def _community_permission_check(action, community, identity):
             "community_id",
             community["processed"][0]["community_id"],
         )
+        community = current_communities.service.read(id_=community_id, identity=identity).to_dict()
```

Comment:
When this is the case, the "community" object passed is a dict that contains the processed entries. The "processed" list contains community-inclusion "request" objects therefore in order to pass it down, seems that we need to query for the community object.

In general this implementation doesn't seem very clean but I guess if we agreed to this in the past we can go with this. An alternative could be to send the community object alongside with the request in the processed dict. Let me know if this good enough or if we should find a better solution.